app/views/directives/notifications/subheading.html

```diff
@@ -0,0 +1 @@
+{{notificationGroup.subHeading}}
```

Comment:
I don't see notificationGroup.subHeading being set anywhere. Could just leave out the subheading-include